audio_context.html

```diff
@@ -0,0 +1,321 @@
+<!doctype html>
+<head>
+  <title>Web Audio Api - Audio Context</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
+<style>
+
+  :root {
+    --flex-direction: column;
+    --line-height: 4em;
+  }
+
+  @media(min-width: 250px) {
+    :root {
+      --flex-direction: row;
+      --line-height: 8em;
+    }
+  }
+
+  h1 {
+    text-align: center;
+  }
+
+  main {
+    outline: thick double;
+    background-color: whitesmoke;
+  }
+
+  fieldset {
+    display: flex;
+    border: none;
+    background: yellow;
+  }
+
+  form > fieldset {
+    flex-direction: var(--flex-direction);
+  }
+
+  fieldset fieldset {
+    background: red;
+  }
+
+  fieldset input[type="range"] {
+    width: 20vw;
+    display: inline-block;
+    top: 0.3em;
+    position: relative;
+  }
+
+  form > fieldset > label {
+    background: turquoise;
+    line-height: var(--line-height);
+  }
+
+  form > header {
+    background: yellow;
+  }
+
+  [type="checkbox"] {
+  }
+
+</style>
+
+</head>
+
+<body>
+
+  <header>
+    <h1>DevPunks Sequencer</h1>
+  </header>
+
+<main>
+
+  <form action=#>
+
+  <header>
+    DevPunks Sequencer
+
+    <label>
+      BPM
+      <input name=bpm value=251 min=10 max=522 type=range>
+    </label>
+
+    <label>
+      Play
+      <input name="Play" type="checkbox">
+    </label>
+  </header>
+
+  <fieldset>
+
+    <legend>Sweep</legend>
+
+    <label>
+      Attack
+      <input name="attack" value="0.2" min="0" max="1" step="0.1" type="range">
+    </label>
+
+    <label>
+      Release
+      <input name="release" value="0.5" min="0" max="1" step="0.1" type="range">
+    </label>
+
+    <fieldset>
+
+      <legend hidden>Checkboxes</legend>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+    </fieldset>
+
+  </fieldset>
+
+  <fieldset>
+
+    <legend>Pulse</legend>
+
+    <label>
+      Hz
+      <input type="range" name="hertz" min="660" max="1320" value="880" step="1">
```

Comment:
May want to give same `name=... value=...` treatment as the rest. No need to change now. technically _zero_  "wrong" with this code. Well done!

Reply:
thanks for pointing it out  ,need to stay consistent with all < input> elements, changing order around